src/signalingclient/DefaultSignalingClient.ts

```diff
@@ -1,6 +1,9 @@
 // Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import * as Crypto from 'crypto';
```

Comment:
Why do we need import all symbols from the crypto-js library (which has a [size](https://www.npmjs.com/package/crypto-js) of 434kb)? Can't we just use standard Web API and avoid taking on this extremely heavy dependency?

https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues